configs/templates/generated_interfaces_for_adapters/list/ListProcessor.go

```diff
@@ -0,0 +1,14 @@
+package list
+
+import "istio.io/mixer/configs/templates/list"
+
+type Instance struct {
+  Blacklist       bool
+  CheckExpression interface{}
+}
+
+type ListProcessor interface {
+  ConfigureList(templateName string, types map[string]*foo_bar_mylistchecker.Type /*typeName to Type mapping*/) error
+  CheckList(templateName string, instances map[string]*Instance /*typeName to Instance (generated from Constructor) mapping*/) (bool, error)
```

Comment:
Why do we include the template name? Aren't we generating one of these interfaces per template? If so, the mapping to template name is implicit, or if you really really need it we could generate a const string with the template name that's private to the package.